### DISCUSSION TITLE
webhook mastodon announce bot

### DISCUSSION THREAD

**Discussion by controlfreakstudio** [owncast/owncast#979] [General Owncast discussion]
Hi team. The directory is awesome and the auto toot it does to its masto bot is super helpful. But rather than just boosting that, here is a masto bot that posts when you go live to your own masto. It's a remix of a weird generative masto bot from the twitter bot, it's quick and dirty and were convensions broken, most assuredly. But it works on a free glitch remix for those that need it. Right now it is grabbing stream name, title, and tags, but from /api/config there is more you can get at if you wish.

I'm a drummer not a writer, but the instructions work with a couple of pastes in <2 minutes. most of the time generating your masto app token and getting your glitch email.

https://glitch.com/edit/#!/owncast-mastodon-announce

**Comment by geekgonecrazy** [comment]
This is awesome!  Thanks for sharing!
